Automattic/jetpack
# Add a check to My Jetpack to determine if a user is "new"

## Proposed changes:

* This diff adds a check in the My Jetpack initialization to determine if a user is "new" to Jetpack
* This data point will be used to vary some of the output of My Jetpack to better suit new versus established users

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Apply this patch to a test site using the Jetpack Beta plugin (the local config has several Jetpack plugins installed already and will not show a user as "new")
* You can check whether the user is new by checking `window.myJetpackInitialState.userIsNewToJetpack` in the browser console, it will be "1" if we consider the user as "new"
* You should be considered a new user until you do one of the following: connect Jetpack, activate a Jetpack module that is not on by default, install another Jetpack plugin, make a Jetpack purchase.
* Test with different combinations of these actions to make sure the property is getting set correctly
